be/src/exprs/vectorized/math_functions.cpp

```diff
@@ -447,7 +447,8 @@ void MathFunctions::decimal_truncate(const int128_t& lv, const int32_t& l_scale,
     }
 }
 
-ColumnPtr MathFunctions::truncate_decimal128(FunctionContext* context, const starrocks::vectorized::Columns& columns) {
+template <DecimalRoundRule rule>
+ColumnPtr MathFunctions::decimal_round(FunctionContext* context, const Columns& columns) {
```

Comment:
Why not improve for const column？

Reply:
Emm, what do you mean? I didn't get it

Reply:
Ignore me. you have improved for const column